database/migrations/2021_11_26_070438_add_index_for_user_ip.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIndexForUserIp extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('activities', function (Blueprint $table) {
+            $table->index('ip', 'ip_address_index');
```

Comment:
Could the index instead be called `activities_ip_index`? Would align better with other indexes and the column name.

Reply:
Sure, I will update the index name.